js/request_test.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import { test, assert, assertEquals } from "./test_util.ts";
+
+test(function initFromRequest(): void {
+  const r = new Request("https://example.com", {
+    body: "ahoyhoy",
+    method: "post"
+  });
+  const req = new Request(r);
+
+  // @ts-ignore
+  assertEquals(req.bodySource, r.bodySource);
+  assertEquals(req.url, "https://example.com");
```

Comment:
Does this actually issue the request? Is the same as fetch? If so, use the test server 

example: https://github.com/denoland/deno/blob/master/js/fetch_test.ts#L29

Reply:
No but I'll definitely add some `fetch` test cases that do!